CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 2.0.0 - UNRELEASED
+
+This version contains is technically a BC break because we drop PHP5 and add return 
+type annotation to support `psr/http-client`. 
+
+### Added
+
+- Support for PSR-18 (Http client).
```

Comment:
Http or HTTP? (i don't really care but we should be consistent in our doc and consistent with how fig documents write it)